orc-rt/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# CMake build for ORC-RT.
+
+#===============================================================================
+# Setup Project
+#===============================================================================
+
+cmake_minimum_required(VERSION 3.20.0)
+
+set(LLVM_COMMON_CMAKE_UTILS ${CMAKE_CURRENT_SOURCE_DIR}/../cmake)
+include(${LLVM_COMMON_CMAKE_UTILS}/Modules/CMakePolicy.cmake
+  NO_POLICY_SCOPE)
+
+project(LibOrcRT LANGUAGES C CXX ASM)
+
+include(GNUInstallDirs)
+
+#===============================================================================
+# Setup CMake Options
+#===============================================================================
+
+option(ORC_RT_INCLUDE_DOCS "Build the ORC-RT documentation." ${ORC_RT_INCLUDE_DOCS})
```

Comment:
What is the default value for this? It looks like it is defaulting to itself, which will be unset on the initial run.